src/bench/bench_bitcoin.cpp

```diff
@@ -80,7 +80,7 @@ int main(int argc, char** argv)
         return EXIT_FAILURE;
     }
 
-    std::unique_ptr<benchmark::Printer> printer = MakeUnique<benchmark::ConsolePrinter>();
+    std::unique_ptr<benchmark::Printer> printer = std::make_unique<benchmark::ConsolePrinter>();
```

Comment:
I'd prefer to defer this stylistic-only scripted diff to avoid merge conflicts

Reply:
Good point. Now fixed. Leaving one use of `std::make_unique` to prove that we're using C++14 :-)